content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -164,7 +164,7 @@ through which the Pod has or has not passed. Kubelet manages the following
 PodConditions:
 
 * `PodScheduled`: the Pod has been scheduled to a node.
-* `PodHasNetwork`: (alpha feature; must be [enabled explicitly](#pod-has-network)) the
+* `PodReadyToStartContainers`: (alpha feature; must be [enabled explicitly](#pod-has-network)) the
```

Comment:
How about a note here stating the condition was named `PodHasNetwork` in k8s 1.25 - 1.27?

Reply:
Added.